elementary/monitor/data_monitoring/data_monitoring.py

```diff
@@ -0,0 +1,113 @@
+import json
+import os.path
+from typing import Any, Dict
+
+import click
+from packaging import version
+
+from elementary.clients.dbt.dbt_runner import DbtRunner
+from elementary.clients.slack.client import SlackClient
+from elementary.config.config import Config
+from elementary.monitor import dbt_project_utils
+from elementary.tracking.anonymous_tracking import AnonymousTracking
+from elementary.utils import package
+from elementary.utils.log import get_logger
+
+logger = get_logger(__name__)
+
+YAML_FILE_EXTENSION = ".yml"
+SQL_FILE_EXTENSION = ".sql"
+
+
+class DataMonitoring:
```

Comment:
Split DataMonitoring into 3:
1. DataMonitoring - base class for data monitoring
2. DataMonitoringAlerts - Responsible for our alerts
3. DataMonitoringReport - Responsible for our report

We got to a point where we have a too much of unnecessary coupling between the report and the alerts.
Those are 2 different features and should not be coupled.

Reply:
No new method, only moved the existing ones to a new file!